### PR TITLE
fix: Project creation failed on GHA fixes #257

### LINKED ISSUE #257: Create new project fails with U-2024.1
UI tests in intellij-openshift-connector fail during initialization, this doesnt fail with U-2023.2/3

```INFO:             Test whether the 'Project SDK' list has loaded all items
      Jul 09, 2024 2:35:56 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:                 Search 'CommonContainerFixture' by '//div[@class='MyDialog']'
      Jul 09, 2024 2:35:56 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:                 Search many 'JPopupMenuFixture's by '//div[@class='HeavyWeightWindow']'
      Jul 09, 2024 2:35:58 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:         waiting(17933ms left)
      Jul 09, 2024 2:35:58 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:             Test whether the 'Project SDK' list has loaded all items
      Jul 09, 2024 2:35:58 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:                 Search 'CommonContainerFixture' by '//div[@class='MyDialog']'
      Jul 09, 2024 2:35:58 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:                 Search many 'JPopupMenuFixture's by '//div[@class='HeavyWeightWindow']'
      Jul 09, 2024 2:35:58 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:     Search many 'JPopupMenuFixture's by '//div[@class='HeavyWeightWindow']'
      Jul 09, 2024 2:35:58 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:     ..click at 'Eclipse Temurin 17.0.11'
      Jul 09, 2024 2:35:59 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO:     ..click at ' /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.11-9/x64'
      Jul 09, 2024 2:36:04 PM com.intellij.remoterobot.stepsProcessing.StepLogger doOnFail
      WARNING:         Failed on step: ..click at ' /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.11-9/x64' (StepWorker.kt_step)
      Jul 09, 2024 2:36:04 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO: Search 'JTextFieldFixture' by '//div[@class='JBTextField']'
      Jul 09, 2024 2:36:04 PM com.intellij.remoterobot.stepsProcessing.StepLogger doBeforeStep
      INFO: Set text 'test-project'
      Jul 09, 2024 2:36:34 PM com.intellij.remoterobot.stepsProcessing.StepLogger doOnFail
      WARNING:     Failed on step: Set text 'test-project' (StepWorker.kt_step)

    GettingStartedTest

      Test initializationError FAILED

      java.net.SocketTimeoutException: timeout
          at okio.SocketAsyncTimeout.newTimeoutException(JvmOkio.kt:146)
          at okio.AsyncTimeout.access$newTimeoutException(AsyncTimeout.kt:161)
          at okio.AsyncTimeout$source$1.read(AsyncTimeout.kt:339)
          at okio.RealBufferedSource.indexOf(RealBufferedSource.kt:430)
          at okio.RealBufferedSource.readUtf8LineStrict(RealBufferedSource.kt:323)
          at okhttp3.internal.http1.HeadersReader.readLine(HeadersReader.kt:29)
          at okhttp3.internal.http1.Http1ExchangeCodec.readResponseHeaders(Http1ExchangeCodec.kt:180)
          at okhttp3.internal.connection.Exchange.readResponseHeaders(Exchange.kt:110)
          at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.kt:93)
          at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
          at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:34)
          at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
          at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:95)
          at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
          at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83)
          at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
          at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:76)
          at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
          at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
          at okhttp3.internal.connection.RealCall.execute(RealCall.kt:154)
          at retrofit2.OkHttpCall.execute(OkHttpCall.java:207)
          at com.intellij.remoterobot.client.IdeRobotClient.execute(IdeRobotClient.kt:63)
          at com.intellij.remoterobot.JavaScriptApi$DefaultImpls.runJs(JavaScriptApi.kt:57)
          at com.intellij.remoterobot.RemoteRobot.runJs(RemoteRobot.kt:32)
          at com.intellij.remoterobot.fixtures.Fixture.runJs(Fixture.kt:92)
          at com.intellij.remoterobot.fixtures.Fixture.runJs$default(Fixture.kt:85)
          at com.intellij.remoterobot.fixtures.JTextFieldFixture$text$2.invoke(JTextFieldFixture.kt:27)
          at com.intellij.remoterobot.fixtures.JTextFieldFixture$text$2.invoke(JTextFieldFixture.kt:26)
          at com.intellij.remoterobot.stepsProcessing.StepWorkerKt.step(StepWorker.kt:23)
          at com.intellij.remoterobot.fixtures.JTextFieldFixture.setText(JTextFieldFixture.kt:26)
          at com.redhat.devtools.intellij.commonuitest.fixtures.dialogs.project.pages.NewProjectFirstPage.setProjectName(NewProjectFirstPage.java:72)
          at com.redhat.devtools.intellij.commonuitest.utils.project.CreateCloseUtils.createNewProject(CreateCloseUtils.java:70)
          at org.jboss.tools.intellij.openshift.test.ui.AbstractBaseTest.setUpProject(AbstractBaseTest.java:67)
          at java.base/java.lang.reflect.Method.invoke(Method.java:568)
          at java.base/java.util.ArrayList.forEach(ArrayList.java:1[511](https://github.com/martinszuc/intellij-openshift-connector/actions/runs/9859031854/job/27221781326#step:6:512))
      Caused by: java.net.SocketException: Socket closed
          at java.base/sun.nio.ch.NioSocketImpl.endRead(NioSocketImpl.java:253)
          at java.base/sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:332)
          at java.base/sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:355)
          at java.base/sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:808)
          at java.base/java.net.Socket$SocketInputStream.read(Socket.java:966)
          at okio.InputStreamSource.read(JvmOkio.kt:93)
          at okio.AsyncTimeout$source$1.read(AsyncTimeout.kt:128)
          ... 32 more
```
          
- 2024.1:
U2024 1

- 2023.3:
U2023 3

### PR DESCRIPTION
**Pull request by martinszuc**
For some reason GHA tests fail during creation of a new project using this library. This click on the name textfield fixes this issue #257

### CODE REVIEW

**Review by richard0202 — APPROVED** [review]
LGTM if tests will pass

File: src/main/java/com/redhat/devtools/intellij/commonuitest/utils/project/CreateCloseUtils.java [hunk 1]
```diff
@@ -67,6 +71,8 @@ public static void createNewProject(RemoteRobot remoteRobot, String projectName,
         newProjectFirstPage.setProjectSdkIfAvailable("17");
 
         if (UITestRunner.getIdeaVersionInt() >= 20221) {
+            remoteRobot.find(ComponentFixture.class, byXpath(XPathDefinitions.JBTEXT_FIELD), Duration.ofSeconds(10)).click(); // Click to gain focus on newProjectFirstPage
```
Comment (lgrossma) [inline src/main/java/com/redhat/devtools/intellij/commonuitest/utils/project/CreateCloseUtils.java:74]: What do you think about adding this as a method to a newProjectFirstPage? Something like `getFocus()` or similar. Would it make sense? 

Note: Not a recommendation, just a genuine question..

Comment (martinszuc) [reply src/main/java/com/redhat/devtools/intellij/commonuitest/utils/project/CreateCloseUtils.java]: Refactored

**Comment by martinszuc** [comment]
I also tried https://github.com/redhat-developer/intellij-common-ui-test-library/commit/940873dc71640155a3d5e632154f352b36708f37 where I added longer duration to previous setProjectName (where the tests failed) but with no result.

**Review by olkornii — APPROVED** [review]
LGTM

**Review by lgrossma — APPROVED** [review]

**Pull request merged**